source/strings.tex

```diff
@@ -3617,7 +3617,7 @@
 \effects
 Equivalent to:
 \begin{codeblock}
-return basic_string_view<charT, traits>(this.data(), pos1, n1).compare(sv);
```

Comment:
OK, I'll prepare a commit removing them all instead.

What dependent base, though? `close` is a member of `basic_filebuf`, not `basic_streambuf`.